### PR TITLE
style: add Pull Request template

### PR DESCRIPTION
**Pull request by YamenMerhi**
# What does this PR introduce?

## 🎨 Style

- [x] Add PR Template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<!-- Thank you for your interest in contributing to LUKSO! -->
+
+<!-- Consider opening an issue for discussion prior to submitting a PR. -->
+<!-- Consider checking CONTRIBUTING.md before contributing. -->
+<!-- New features will be merged faster if they were first discussed and designed with the team. -->
```
Comment (CJ42) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Maybe we can suggest to either:
- open an issue.
- go to the discussion tab of the repo?

first?

image

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Fixes #???? <!-- Fill in with issue number -->
+
+<!-- Describe the changes introduced in this pull request here. -->
+
+<!-- Include any context necessary for understanding the PR's purpose. (Images, links, etc ..) -->
+
+### PR Checklist
+
+<!-- Before merging the pull request all of the following must be complete. -->
```
Comment (CJ42) [inline .github/PULL_REQUEST_TEMPLATE.md:60]:
```suggestion
<!-- Before merging the pull request, making sure you have run locally the following. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+<!-- Feel free to submit a PR or Draft PR even if some items are pending. -->
+<!-- Some of the items may not apply. -->
```
Comment (CJ42) [inline .github/PULL_REQUEST_TEMPLATE.md:62]:
```suggestion
<!-- (Some of the items may not apply.) -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+- [ ] Tests
```
Comment (CJ42) [inline .github/PULL_REQUEST_TEMPLATE.md:64]: Is it what was meant here?

```suggestion
- [ ] Wrote Tests
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+- [ ] Documentation
+- [ ] Ran `npm run test`
+- [ ] Ran `npm run linter`
+- [ ] Ran `npm run prettier`
```
Comment (CJ42) [inline .github/PULL_REQUEST_TEMPLATE.md:68]: I suggest we put them in the same order as the CI:

- [ ] Ran `npm run linter`
- [ ] Ran `npm run prettier`
- [ ] Ran `npm run build`
- [ ] Ran `npm run test`

image

**Review by CJ42 — APPROVED** [review]
Very great idea! This PR template looks great! 🥳 ⭐

**Review by b00ste — APPROVED** [review]

**Review by skimaharvey — APPROVED** [review]

**Pull request merged**